### PR TITLE
feat: Add asset packs owner query parameter

### PR DESCRIPTION
**Pull request by LautaroPetaccio**
This PR creates two new endpoints:
- The `/assetPacks/default` endpoint, that returns the default asset packs.
- The `/ownAssetPacks` authenticated endpoint, that returns the user's asset packs.

### CODE REVIEW

File: src/AssetPack/AssetPack.router.ts [hunk 1]
```diff
@@ -100,22 +100,43 @@ export class AssetPackRouter extends Router {
...
     const ethAddress = req.auth ? req.auth.ethAddress : ''
-    let assetPacks: FullAssetPackAttributes[] = []
+    const owner = req.query.owner
     const tracer = uuidv4()
     this.logger.info(
       `Starting request to get the assets packs with tracer ${tracer} and the address "${ethAddress}"`
     )
 
+    // Process the owner query parameter first
+    if (owner && owner === 'default') {
```
Comment (nachomazzara) [inline src/AssetPack/AssetPack.router.ts:110]: I think that `owner` here is redundant.

File: src/AssetPack/AssetPack.router.ts [hunk 2]
```diff
@@ -100,22 +100,43 @@ export class AssetPackRouter extends Router {
...
+      return this.logExecutionTime(
+        this.retrieveDefaultAssetPacks,
+        'Get default asset packs',
+        tracer
+      )
+    } else if (owner && owner === ethAddress) {
```
Comment (nachomazzara) [inline src/AssetPack/AssetPack.router.ts:116]: I think that `owner` here is redundant.

File: src/AssetPack/AssetPack.router.ts [hunk 3]
```diff
@@ -100,22 +100,43 @@ export class AssetPackRouter extends Router {
 
   getAssetPacks = async (req: AuthRequest) => {
     const ethAddress = req.auth ? req.auth.ethAddress : ''
-    let assetPacks: FullAssetPackAttributes[] = []
+    const owner = req.query.owner
```
Comment (nachomazzara) [inline src/AssetPack/AssetPack.router.ts:103]: I would lowercase `req.query.owner` to sopport checksummed addresses too

Comment (nachomazzara) [reply src/AssetPack/AssetPack.router.ts:103]: WE can do it in a reutilizable middleware

**Review by nachomazzara — CHANGES_REQUESTED** [review]
LGTM, just minor comments

File: src/AssetPack/AssetPack.router.ts (hunk 3 above)
Comment (LautaroPetaccio) [reply src/AssetPack/AssetPack.router.ts:103]: I'll lowercase it!
How would the middleware be? Will it just lowercase an address? Do we need it somewhere else?

File: src/AssetPack/AssetPack.router.ts (hunk 1 above)
Comment (LautaroPetaccio) [reply src/AssetPack/AssetPack.router.ts:110]: Your right, I'll remove it.

**Review by nachomazzara — APPROVED** [review]
LGTM

**Pull request merged**